bin/northamptonshire/update_ids_for_v2

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env perl
+
+# The migration from Alloy V1 to V2 causes the IDs to change. This takes a CSV
+# with a mapping of old ids to new ones and applies it to the external_id, saving
+# the old one in extra in case it's relevant.
+
+use strict;
+use warnings;
+use v5.14;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../setenv.pl";
+}
+
+use FixMyStreet::DB;
+
+use Getopt::Long;
+use Text::CSV;
+
+my $NEW_ID = 'ALLOY_ITEM_ID';
+my $OLD_ID = 'RES_ID';
+
+my ($commit, $file);
+GetOptions(
+    'commit' => \$commit,
+    'file=s' => \$file,
+);
+
+if (!$commit) {
+    say "*** DRY RUN ***";
+}
+
+my $csv = Text::CSV->new({ binary => 1 });
+open my $fh, "<:encoding(utf-8)", $file or die "Failed to open $file: $!\n";
+my @cols = @{$csv->getline($fh)};
+$csv->column_names (@cols);
+
+my $count = 0;
+my $northants = FixMyStreet::DB->resultset("Body")->find({ name => 'Northamptonshire County Council' });
+if ($northants) {
+    while (my $report = $csv->getline_hr( $fh ) ) {
+        my $p = FixMyStreet::DB->resultset('Problem')->search({ external_id => $report->{$OLD_ID}, bodies_str => $northants->id })->first;
```

Comment:
Probably want to use `->to_body(...)` rather than direct bodies_str lookup in case there's a report sent to both county and district (doesn't look like there is though).